src/core/Expander/ExpanderTitle/ExpanderTitle.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { Component, ReactNode } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { withSuomifiDefaultProps } from '../../theme/utils';
+import { TokensProp, InternalTokensProp } from '../../theme';
+import {
+  HtmlDiv,
+  HtmlButton,
+  HtmlButtonProps,
+  HtmlDivProps,
+} from '../../../reset';
+import { expanderTitleBaseStyles } from './ExpanderTitle.baseStyles';
+import { Icon } from '../../Icon/Icon';
+import { ExpanderConsumer, ExpanderTitleBaseProps } from '../Expander/Expander';
+import { VisuallyHidden } from '../../../components';
+
+const baseClassName = 'fi-expander';
+const titleClassName = `${baseClassName}_title`;
+const titleOpenClassName = `${titleClassName}--open`;
+const titleButtonClassName = `${titleClassName}-button`;
+const iconClassName = `${titleClassName}-icon`;
+const iconOpenClassName = `${iconClassName}--open`;
+
+export interface ExpanderTitleProps extends Omit<HtmlDivProps, 'classname'> {
+  /** Custom classname to extend or customize */
+  className?: string;
+  /** Title for Expander */
+  children?: ReactNode;
+  /** Additional text for closed expanders toggle button. E.g."open expander". */
+  ariaOpenText: string;
+  /** Additional text for open expanders toggle button. E.g."close expander". */
+  ariaCloseText: string;
+  /** Properties for title open/close toggle button */
+  toggleButtonProps?: Omit<
+    HtmlButtonProps,
+    | 'onClick'
+    | 'onMouseDown'
+    | 'onMouseUp'
+    | 'onKeyPress'
+    | 'onKeyUp'
+    | 'onKeyDown'
+  >;
+}
+
+export interface InternalExpanderTitleProps
+  extends ExpanderTitleProps,
+    ExpanderTitleBaseProps {}
+
```

Comment:
Are there need to export these internal props? (_same with ExpanderTitleButton_)

Reply:
Probably not, those were reused originally and were probably left there by accident. Will check and remove is possible.

Reply:
Exports removed.